.circleci/config.yml

```
@@ -943,9 +946,12 @@ jobs:
                       name: Restore cached uberjar built in previous step
                       <<: *CacheKeyUberjar
                   - steps: << parameters.before-steps >>
-                # Make both `test-files` and `source-folder` parameters optional. Translates to: if `parameter` => run associated flag (`--spec` and `--folder`, respectively)
+                # Make both `test-files`,  `source-folder` and `currents-record` parameters optional. Translates to: if `parameter` => run associated flag (`--spec`, `--folder` and `--key $CURRENTS_KEY --record` respectively)
                 command: |
-                  run test-cypress-no-build <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >> <<# parameters.source-folder >> --folder << parameters.source-folder >> <</ parameters.source-folder >>
+                  run test-cypress-no-build \
```

Are you sure we need backslashes at the end of each row?
```suggestion
                  run test-cypress-no-build
```

---

But isn't the pipe next to command exactly the pointer that says: "Hey, interpret next lines as multi-line command"?
`command: |`

---

As I understand, every time we run `command` CCI runs on a shell. 

In a shell script, if we are doing multiple lines in one command we need to properly add the `\` to it to understand that the command continues. If not, it understands as one command per line.